packages/react/test/specs/components/Accordion/AccordionTitle-test.tsx

```diff
@@ -8,26 +6,17 @@ describe('AccordionTitle', () => {
     eventTargets: {
       onClick: `.${AccordionTitle.slotClassNames.content}`,
     },
-    requiredProps: {
-      contentRef: _.noop,
-    },
   })
 
   describe('accessiblity', () => {
     describe('header', () => {
       handlesAccessibility(AccordionTitle, {
-        requiredProps: {
-          as: 'h3',
-          contentRef: _.noop,
-        },
+        requiredProps: { as: 'h3' },
```

Comment:
@silviuavram are these changes correct? also why is `requiredProps: { as: 'h3' },` needed?

Reply:
if it's a header it will not receive a role. you can see that the example below does not have the as prop as required, but the `handlesAccessibility` test will check that the role of `header` has been added.

Reply:
got it, so the changes are good, thanks